docs/versioning/android-version.md

```diff
@@ -81,32 +81,36 @@ If true, omits zero in the patch version. So _42.10.0_ will become _42.10_ and _
 
 This component works on standard build.gradle files. If you use flavors in your build.gradle, you can set the flavor name. However, please be aware that flavor support is not foolproof. Due to dynamic nature of build.gradle file, it may not cover all the cases.
 
-### Output Values
+### Output Variables
```

Comment:
Wouldn't it be better if this part comes after "input variables"?
```suggestion
### Output Variables
```

Reply:
Since the topics under the Input heading are very long, so I decided to preserve the system was before.